AutoCollection/Performance.ts

```diff
@@ -135,10 +137,26 @@ class AutoCollectPerformance {
                 totalIrq += irq;
             }
 
+            // Calculate % of total cpu time (user + system) this App Process used (Only supported by node v6.1.0+)
+            let appCpuPercent: number = undefined;
+            if (typeof (process as any).cpuUsage === 'function') {
+                const appCpuUsage = (process as any).cpuUsage();
+                const hrtime = process.hrtime();
+
+                const totalApp = ((appCpuUsage.user - this._lastAppCpuUsage.user) + (appCpuUsage.system - this._lastAppCpuUsage.system)) || 0;
+                const elapsedTime = ((hrtime[0] - this._lastHrtime[0])*1e6 + (hrtime[1] - this._lastHrtime[1])/1e3) || 0; // convert to microseconds
```

Comment:
Should we if guard this with the existence of `lastHrtime`?

Reply:
added a guard for both calculations